examples/webgpu_compute_cloth.html

```diff
@@ -0,0 +1,558 @@
+
+	import { GUI } from 'three/addons/libs/lil-gui.module.min.js';
+	import { OrbitControls } from 'three/addons/controls/OrbitControls.js';
+	import { RGBELoader } from 'three/addons/loaders/RGBELoader.js';
+
+	let renderer, scene, camera, controls;
+
+	const clothWidth = 1;
+	const clothHeight = 1;
+	const clothNumSegmentsX = 30;
+	const clothNumSegmentsY = 30;
+	const sphereRadius = 0.15;
+
+	let vertexPositionBuffer, vertexForceBuffer, vertexSpringCountBuffer, vertexSpringPointerBuffer, vertexIsFixedBuffer;
+	let springVertexIdBuffer, springRestLengthBuffer, springForceBuffer;
+	let springListBuffer;
+	let computeSpringForces, computeVertexForces;
+	let dampeningUniform, spherePositionUniform, stiffnessUniform, sphereUniform, windUniform;
+	let vertexWireframeObject, springWireframeObject;
+	let clothMesh, clothMaterial, sphere;
+	let timeSinceLastStep = 0;
+	let timestamp = 0;
+	const verletVertices = [];
+	const verletSprings = [];
+	const verletVertexColumns = [];
+
+	const clock = new THREE.Clock();
+
+	const params = {
+		wireframe: false,
+		sphere: true,
+		wind: 1.0,
+	};
+
+	init();
+
+	async function init() {
+
+		renderer = new THREE.WebGPURenderer( { antialias: true } );
+		renderer.setPixelRatio( window.devicePixelRatio );
+		renderer.setSize( window.innerWidth, window.innerHeight );
+		renderer.setAnimationLoop( render );
+		renderer.toneMapping = THREE.ACESFilmicToneMapping;
+		renderer.toneMappingExposure = 1.35;
+		document.body.appendChild( renderer.domElement );
+
+		scene = new THREE.Scene();
+
+		camera = new THREE.PerspectiveCamera( 40, window.innerWidth / window.innerHeight, 0.01, 10 );
+		camera.position.set( - 1.6, - 0.1, - 1.6 );
+
+		controls = new OrbitControls( camera, renderer.domElement );
+
+		controls.target.set( 0, - 0.1, 0 );
+		controls.minDistance = 1;
+		controls.maxDistance = 3;
+		controls.addEventListener( 'change', render );
+
+
+		const rgbeLoader = new RGBELoader().setPath( 'textures/equirectangular/' );
+
+		const hdrTexture = await rgbeLoader.loadAsync( 'royal_esplanade_1k.hdr' );
+		hdrTexture.mapping = THREE.EquirectangularReflectionMapping;
+		scene.background = hdrTexture;
+		scene.backgroundBlurriness = 0.5;
+		scene.environment = hdrTexture;
+
+		setupCloth();
+
+		const gui = new GUI();
+		gui.add( stiffnessUniform, 'value', 0.1, 0.5, 0.01 ).name( 'stiffness' );
+		gui.add( params, 'wireframe' );
+		gui.add( params, 'sphere' );
+		gui.add( params, 'wind', 0, 5, 0.1 );
+		const materialFolder = gui.addFolder( 'material' );
+		materialFolder.addColor( clothMaterial, 'color' );
+		materialFolder.add( clothMaterial, 'roughness', 0.0, 1, 0.01 );
+		materialFolder.add( clothMaterial, 'metalness', 0.0, 1, 0.01 );
+		materialFolder.add( clothMaterial, 'sheen', 0.0, 1, 0.01 );
+		materialFolder.add( clothMaterial, 'sheenRoughness', 0.0, 1, 0.01 );
+		materialFolder.addColor( clothMaterial, 'sheenColor' );
```

Comment:
Sheen is not intended for metals.

I'd only include menu items that can be freely-changed independently. Otherwise, the demo appears to be broken.